src/Mono.Android/Android.Runtime/JNINativeWrapper.cs

```diff
@@ -13,13 +13,16 @@ public static class JNINativeWrapper {
 
 		static void get_runtime_types ()
 		{
+#if MONOANDROID1_0
 			if (mono_unhandled_exception_method != null)
 				return;
-#if MONOANDROID1_0
 			mono_unhandled_exception_method = typeof (System.Diagnostics.Debugger).GetMethod (
 				"Mono_UnhandledException", BindingFlags.NonPublic | BindingFlags.Static);
 			if (mono_unhandled_exception_method == null)
 				AndroidEnvironment.FailFast ("Cannot find System.Diagnostics.Debugger.Mono_UnhandledException");
+#else
+			if (exception_handler_method != null)
```

Comment:
Should this be made the "primary" "is initialized" check, instead of checking for `mono_unhandled_exception_method` under `MONOANDROID1_0` and `exception_handler_method` for NET5?

```csharp
static void get_runtime_types ()
{
	if (exception_handler_method != null)
		return;
#if MONOANDROID1_0
	mono_unhandled_exception_method = typeof (System.Diagnostics.Debugger).GetMethod (…)
	…
#endif  // MONOANDROID1_0

	exception_handler_method = typeof (AndroidEnvironment).GetMethod (…);
	…
}
```